### ISSUE TITLE
Library won't load, videos won't play

### ISSUE THREAD

**Issue by 16434** [plexinc/plex-for-kodi#379]
## Details:
- Add-on version: 0.3.4
- Kodi version: OSMC 2022.09-1
- OS Platform: Ubuntu
- Hardware: RPi3

## Checklist
- [ ] I have included a link to a log (or at the very least a link to a forum post with a log) from a session that had the issue
- [ ] I have added appropriate GitHub labels

**Comment by crxm** [comment]
I updated OSMC last night and ran into the same issue. I found this thread on the OSMC forums which discusses the issue:

https://discourse.osmc.tv/t/september-update-killed-plex-on-my-vero-4k/93651

According to the thread, it looks like this addon relies on a python command (**thread.isAlive()**) which has been deprecated in the more recent version of python provided by the update.

A patched version of the addon was posted by a user in the thread which has been reported to resolve the issue, although I have no tested this myself yet. It looks like updates have been made to replace the deprecated command in the develop branch back in April, as seen in this commit:

https://github.com/plexinc/plex-for-kodi/commit/984a572248fe5c1af7451dc3d36a3e7bd82c6dd9

Is a new release planned which would include this fix in the near future?
